security/pkg/nodeagent/plugin/providers/google/stsclient/stsclient.go

```diff
@@ -106,12 +106,12 @@ func (p Plugin) ExchangeToken(ctx context.Context, credFetcher security.CredFetc
 	respData := &federatedTokenResponse{}
 	if err := json.Unmarshal(body, respData); err != nil {
 		return "", time.Now(), resp.StatusCode, fmt.Errorf(
-			"failed to unmarshal response data. HTTP status: %s. Error: %v. Body size: %d", resp.Status, err, len(body))
+			"failed to unmarshal response data. HTTP status: %s. Req: %s. Error: %v. Body size: %d", resp.Status, string(jsonStr), err, len(body))
```

Comment:
Don't print out the entire request. There are sensitive information like JWT in the request string.

Reply:
Ok.